### PR TITLE
style: add message rtl

### LINKED ISSUE #22473: Icon of message in RTL
- [yes ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/antd-reproduction-template-53okh)

### Steps to reproduce
Click on the  button in the [code sandbox](https://codesandbox.io/s/antd-reproduction-template-53okh)

### What is expected?
the icon of message should be on the right

### What is actually happening?
the icon of message is on the left

| Environment | Info |
|---|---|
| antd | 4.0.3 |
| React | 16.13.0 |
| System | windows 10 |
| Browser | chrome 80 |

---
check antd notification (maybe has the same problem)

### PR DESCRIPTION
**Pull request by xrkffgg**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/22473

### 💡 Background and solution

用 `ConfigConsumer` 嵌套 Message，试了半天没成功，所以用了这种方式。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Add Message RTL mode        |
| 🇨🇳 Chinese |  新增 Message RTL 模式         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/message/demo/other.md](https://github.com/ant-design/ant-design/blob/add-rtl-message/components/message/demo/other.md)
[View rendered components/message/index.en-US.md](https://github.com/ant-design/ant-design/blob/add-rtl-message/components/message/index.en-US.md)
[View rendered components/message/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/add-rtl-message/components/message/index.zh-CN.md)

### CODE REVIEW

**Comment by saeedrahimi** [comment]
@xrkffgg thanks for your great effort on RTL of antd.
Honestly i'm not ok with this solution and hard-coding direction of `message`.
In multi-language projects (one language is LTR and another is RTL) switching between languages and direction will be hard with this method.
I know there is no `ConfigContext` in `message` component, but there should be a solution from creators.
@afc163 , @zombieJ Could you help on this.?

**Comment by yoyo837** [comment]
Agree with @saeedrahimi

**Comment by xrkffgg** [comment]
I agree and I know this is not a good solution.

**Comment by afc163** [comment]
```jsx
message.config({
  rtl: true,
})
```

How about global config for `message.xxx`.

**Comment by xrkffgg** [comment]
I have changed, how about this?
@afc163 @saeedrahimi @yoyo837

File: components/message/index.tsx [hunk 1]
```diff
@@ -155,6 +159,9 @@ const api: any = {
       maxCount = options.maxCount;
       messageInstance = null;
     }
+    if (options.rtl !== undefined) {
```
Comment (yoyo837) [inline components/message/index.tsx]: in ?

Comment (xrkffgg) [reply components/message/index.tsx]: 啥意思？

Comment (yoyo837) [reply components/message/index.tsx]: 'rtl' in options 就覆盖比较符合常规理解。

Comment (xrkffgg) [reply components/message/index.tsx]: 我是按照 代码 之前的方式，之前都是用的这种方式

Comment (zombieJ) [reply components/message/index.tsx]: `!== undefined` ，免得用户传了一个 `rlt = undefined`。之前有几个 issue 就是这个问题。

**Review by saeedrahimi — APPROVED** [review]

**Pull request merged**